README.md

```diff
@@ -114,6 +114,18 @@ By default, ```sigh``` will install the downloaded profile on your machine. If y
 
     sigh --skip_install
 
+If you need the provisioning profile to be renewed regardless of it's state use the --force option. This gives you a profile with the maximum lifetime:
+
+    sigh --renew --force -a com.krausefx.app -u username
```

Comment:
Whoops. That should actually be just renew instead of --renew...
--force without the renew just downloads the profile.